docs/pages/access-controls/guides/headless.mdx

```diff
@@ -178,6 +179,26 @@ alice@server01 $
   your local terminal.
 </Notice>
 
+## Optional: Teleport Connect
+
+Teleport Connect v13.3.1+ can also be used to approve headless logins seamlessly.
+Teleport Connect will automatically detect the headless login attempt and display
+the details in the App for you to `Approve` or `Cancel`.
+
+<Figure width="700">
+![Headless Confirmation](../../../img/headless/confirmation.png)
+</Figure>
+
+When you hit `Approve`, you will be prompted to tap your MFA key to complete the approval
```

Comment:
I think we're overusing `Code Formatting`. Approve is just a verb, we don't need any special formatting.

```suggestion
You will be prompted to tap your MFA key to complete the approval
```